doc/gui/tutorial_erp.ipynb

```diff
@@ -220,7 +234,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "gui._simulate_left_tab_click(\"Network connectivity\")"
+    "gui._simulate_left_tab_click(\"Network\")"
```

Comment:
I think we're going to move away from the automatically generated GUI documentation so not necessary to update this file (no need to remove this change of course)

Reply:
I think one of the tests fails if the 'Network Connectivity' tab title changes but it's not updated in the notebook.